client/fingerprint/host.go

```diff
@@ -29,12 +31,21 @@ func (f *HostFingerprint) Fingerprint(req *FingerprintRequest, resp *Fingerprint
 		return err
 	}
 
+	if runtime.GOOS == "windows" {
+		platformVersion := strings.Split(hostInfo.PlatformVersion, "Build")
+		if len(platformVersion) == 2 {
+			resp.AddAttribute("os.version", strings.TrimSpace(platformVersion[0]))
+			resp.AddAttribute("os.build", strings.TrimSpace(platformVersion[1]))
+		} else {
+			f.logger.Warn(fmt.Sprintf("unable to retrieve 'os.build' attribute %s", platformVersion))
```

Comment:
The hclog library gives us key-value pairs for the extra info we might want to fill out at runtime. And this gives us the original text rather than the post-parsed text:

```suggestion
			f.logger.Warn("unable to retrieve 'os.build' attribute", "platform_version", hostInfo.PlatformVersion))
```